pages/en/lb3/Authentication-authorization-and-permissions.md

```diff
@@ -45,15 +35,196 @@ LoopBack's access control system is built around a few core concepts, as summari
 
 The general process to implement access control for an application is:
 
-1.  **Specify user roles**.
+1.  **Implement authentication**:
+    In the application, add code to create (register) new users, login users (get and use authentication tokens), and logout users.
+2.  **Specify user roles**.
     Define the user roles that your application requires.
-    For example, you might create roles for anonymous users, authorized users, and administrators. 
-2.  **Define access for each role and model method**.
+    For example, you might create roles for anonymous users, authenticated users, group members, and administrators. 
+3.  **Define access for each role and model method**.
     For example, you might enable anonymous users to read a list of banks, but not allow them to do anything else.
     LoopBack models have a set of built-in methods, and each method maps to either the READ or WRITE access type.
-    In essence, this step amounts to specifying whether access is allowed for each role and each Model - access type, as illustrated in the example below.
-3.  **Implement authentication**:
-    in the application, add code to create (register) new users, login users (get and use authentication tokens), and logout users.
+    In essence, this step amounts to specifying whether access is allowed for each role and each Model - access type, as illustrated in the example below.<br>
+		_NOTE: you can also map directly access rights to specific users or applications._
+4.  **Set-up access control for users**:
+		This can be achieved either by statically mapping users with any role you may have earlier created using the Role model, **or** by adding code to register dynamic role resolvers that will at runtime resolve whether a user, given a preconfigured set of conditions, has a given role.
+
+## Initial setup
+
+### Enabling Access control
+
+When you create your app with the LoopBack [application generator](Application-generator.html), access control is automatically enabled, _except_ if you choose the "empty-server" application type.
+To enable access control for an "empty-server" application, you must add a boot
+script that calls `enableAuth()`. For example, in `server/boot/authentication.js`:
+
+```js
+module.exports = function enableAuthentication(server) {
+  server.enableAuth();
+};
+```
```

Comment:
@bajtos : then i'll had a tip in case the user opts for the multiple user models implementation : models to attach should then be defined in `model-config.json`, and `enableAuth()` should be called without the  `{ dataSource: 'db' }` argument. I suspect leaving it would mess things.